_includes/warning.html

```diff
@@ -0,0 +1,7 @@
+<div>
+  <aside class="warning" id="wc-warning">
+    <p>
+      Trying to get a wildcard certificate?
+      Please use the dropdown menus below to get instructions specific to your system, and read those instructions carefully.
```

Comment:
Are we missing the closing `</p>` tag here or was this intentional?

Reply:
Not intentional, thanks for catching that!